ts/features/lollipop/saga/index.ts

```diff
@@ -61,6 +73,37 @@ export function* deleteCurrentLollipopKeyAndGenerateNewKeyTag() {
   yield* put(lollipopKeyTagSave({ keyTag: newKeyTag }));
 }
 
+export function* lollipopKeyCheckWithServer() {
```

Comment:
Is `lollipopKeyCheckWithServer` providing enough information on what the function does? Would not something like `checkLollipopSessionAssertionAndInvalidateIfNeeded` be better?